src/components/ErrorPage/ErrorPage.js

```diff
@@ -32,55 +32,77 @@ const styles = theme => ({
 	},
 	pageContainer: {
 		minHeight: '100vh'
+	},
+	hiddenImage: {
+		textAlign: 'center',
+		marginTop: theme.spacing(4)
 	}
-
 });
 
 
 function ErrorPage({ classes }) {
 	const data = useStaticQuery(graphql`
-      query {
-        file(relativePath: {eq: "404/bowimg-removebg.png"}) {
-            childImageSharp {
-                fixed(height: 400) {
-                  base64
-                  width
-                  height
-                  src
-                  srcSet
-                }
+  query {
+    file(relativePath: {eq: "404/bowimg-removebg.png"}) {
+        childImageSharp {
```

Comment:
You are using 2 spaces between the `query` line and the `file` line, you should be using 2 spaces between the `file` line and the `childImageSharp` line